adserver/views.py

```diff
@@ -258,6 +260,47 @@ def get_object(self, queryset=None):  # pylint: disable=unused-argument
         )
 
 
+class FlightUpdateView(StaffAccessMixin, UserPassesTestMixin, UpdateView):
```

Comment:
Should the StaffAccessMixin inhereit from the UserPassesTestMixin to make sure we don't miss that part when re-using the code?

Reply:
Not only should it, but we already had a version of this mixin. We just need to define and use it in one place.